src/Tribe/Venue.php

```diff
@@ -495,16 +495,19 @@ public function update( $venue_id, $data ) {
 		                      ) );
 
 		if ( count( $args ) > 1 ) {
+			$tag = 'save_post_' . Tribe__Events__Main::VENUE_POST_TYPE;
```

Comment:
I think it would be nice if we could build the tag with a variable in the string. Normally easy to regexp for.